sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@
 
 ### Features Added
 - This version of the client library marks a beta release and defaults to the service API version `2022-10-01-preview`.
+- Added properties `fhirVersion` and `documentType` to `AnalyzeHealthcareEntitiesOptions` and `AnalyzeHealthcareEntitiesAction`.
+  Use the keyword to indicate the version for the `fhirBundle` contained on the `AnalyzeHealthcareEntitiesResult`.
```

Comment:
Is "keyword" the right terminology in Java?

Reply:
Goot catch. I would remove it.